README.md

```diff
@@ -28,7 +28,7 @@ result faceting and highlighting.
 ## Online Documentation
 
 This README file only contains basic setup instructions.  For more
-comprehensive documentation, visit <https://solr.apache.org/guide/>
+comprehensive documentation, visit <https://solr.apache.org/guide/solr>
```

Comment:
I tested, and `https://solr.apache.org/guide/` works, `https://solr.apache.org/guide/solr` doesn't....

Reply:
However, https://solr.apache.org/guide/solr/latest/configuration-guide/solr-plugins.html works, and https://solr.apache.org/guide/latest/configuration-guide/solr-plugins.html doesn't, so maybe this is okay?

Reply:
Nevermind, I see you made a change later on to make the urls have /solr/ ;-)

Reply:
Yeah antora comes with a default htaccess rule to redirect `/guide` to `/guide/solr/latest/index.html`. For some reason it doesn't also redirect `/guide/solr` to `/guide/solr/latest/index.html`. So I'm going to add that to solr site as well.